### DISCUSSION TITLE
Retranslator servers are down in Russia for more than 24 h

### DISCUSSION THREAD

**Discussion by apawlow** [rustdesk/rustdesk#5254] [General]
Retranslator servers are down in [Russia for more than 24 h](https://downradar.ru/ne-rabotaet/rustdesk.com) - is this a kind of permanent and I shall look fort alternative or you are fixin' it?

**Comment by dinger1986** [comment]
I don't know the answer but why don't you deploy your own server?

**Reply by marvellz** [reply]
I don't think it's necessary for private use.

**Comment by apawlow** [comment]
thank you, will think of it

пт, 4 авг. 2023 г. в 21:28, dinger1986 ***@***.***>:

**Reply by marvellz** [reply]
@apawlow it is observed at some providers. Contact your ISP for support.
There are no problems with small and not popular ISP.

**Reply by surfining** [reply]
С конченным Ростелеком не работает!


Свой сервер разварачивается на своём Синолоджи за 15 минут или на арендованном
